docs/README.md

```diff
@@ -5,7 +5,7 @@ They are published on powsybl.readthedocs.io/ and pull requests are built and pr
 
 To build the docs locally, run the following commands:
 ~~~bash
-pip install -r docs/requirements.txt
-sphinx-build -a docs ./build-docs
+pip install -r requirements.txt
+sphinx-build -a . ../build-docs
 ~~~
-Then open `build-docs/index.html` in your browser.
\ No newline at end of file
+Then open `../build-docs/index.html` in your browser.
```

Comment:
```suggestion
Then open `build-docs/index.html` in your browser.
```
Same here.

I think that the reason you add problems launching your commands locally is that you were in the "wrong" directory (i.e. the `docs` directory instead of being in `powsybl-core`).
Should we explain that better?

Reply:
The issue is the location of your README.md file for me.

Reply:
The README.md is about the documentation so I think its location makes sense.

Maybe we can add the sentence you can find in the powsybl-open-rao docs/README.md to make it less ambiguous:

When modifying the website content, you can easily preview the result on your PC **by navigating to the root of the project** and running:

pip install -r docs/requirements.txt
sphinx-build -a docs ./build-docs

Then open build-docs/index.html in your browser.

Reply:
Ok, now I get the problem when running the commands on an IDE.

I will amend the README.md so that the commands can be run from an IDE too.